mirrord/config/src/feature/network/incoming/http_filter.rs

```diff
@@ -12,6 +12,7 @@ use crate::{
 // TODO(alex) [high] 2023-05-18: Wrong example (incomplete) here and in the oficial docs.
 /// Filter configuration for the HTTP traffic stealer feature.
 ///
+/// DEPRECATED - USE http_filter instead, unless using old operator/agent version (pre 3.46.0)
```

Comment:
I'd add `#[config(deprecated)]` to `filter` and `ports` to show warning in runtime

Reply:
I tried to do it to the root object, (incomingconfig) but that didn't work, good suggestion